source/joker.c

```diff
@@ -12,17 +12,17 @@
 #include <string.h>
 #include <tonc.h>
 
-#define JOKER_SCORE_TEXT_Y         48
-#define HELD_CARD_SCORE_TEXT_Y     108
-#define MAX_CARD_SCORE_STR_LEN     2
-#define NUM_JOKERS_PER_SPRITESHEET 2
+#define JOKER_SCORE_TEXT_Y     48
+#define HELD_CARD_SCORE_TEXT_Y 108
+#define MAX_CARD_SCORE_STR_LEN 2
+#define MAX_NUM_JOKERS_SPRITESHEETS \
+    75 // what it was before (MAX_DEFINEABLE_JOKERS / JOKERS_PER_SPRITESHEET)
```

Comment:
Clang-format is trying to fit the trailing comment on one line with the column limit. Putting the comment above should help.